### PR TITLE
WRR-977: Fixed Scroller with editable prop to move items or complete edit mode when keydown event fired in pointer mode

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
When selecting an item in an editable scroller and pressing the 5-way keys (e.g. Left, Right, Enter, Down) in pointer mode, the item does not move or the edit mode does not complete.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
When 5-way key is pressed in pointer mode, only `handleGlobalkeyDownCapture` function is called. So I added logic in global keydown handler.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
More details are available at https://github.com/enactjs/sandstone/pull/1657.

### Links
[//]: # (Related issues, references)
WRR-977
https://github.com/enactjs/sandstone/pull/1657

### CODE REVIEW

File: samples/sampler/stories/qa/IconItem.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import {add, is} from '@enact/core/keymap';
 import Button from '@enact/sandstone/Button';
 import IconItem from '@enact/sandstone/IconItem';
 import Scroller from '@enact/sandstone/Scroller';
+import Spotlight from '@enact/spotlight';
 import $L from '@enact/sandstone/internal/$L';
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/IconItem.js:6]:
```suggestion
import $L from '@enact/sandstone/internal/$L';
import Spotlight from '@enact/spotlight';
```

**Comment by MikyungKim** [comment]
Please resolve the merge conflict.

File: samples/sampler/stories/qa/IconItem.js (hunk 1 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/IconItem.js]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]

**Pull request merged**